### PR TITLE
Updating the README.md file

### PR DESCRIPTION
**Pull request by JuanCab**
This PR consists of edits to the README.md file which are meant to:
- Provide better documentation as to the purpose of `stellarphot`
- Provide slightly expanded installation instructions (Addressing #333) to clarify how to handle the needed packages for exoplanet transit light curves.  

I have not added instructions for installing `stellarphot` for developers for now.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,48 +1,54 @@
...
 
 ## Installation
 
-You can install `stellarphot` with either pip or conda. `conda` is recommended at the moment because of an issue with installing one of the dependencies of `stellarphot` in Python 3.9 or higher
+You can install `stellarphot` with either `pip` or `conda`.  If you are interested in `stellarphot` for exoplanet transit light curves, `conda` is recommended at the moment because of an issue with installing one of the dependencies.
+
++ Install with `conda`:
+
+  	conda install -c conda-forge stellarphot reducer
+  	pip install astronbs
```
Comment (mwcraig) [inline README.md:20]: This needs to be wrapped in triple backticks to display as code. Indenting only works for rst.

File: README.md [hunk 2]
```diff
@@ -1,48 +1,54 @@
...
-or install with `pip`:
+  	pip install stellarphot reducer astronbs
 
-```
-pip install stellarphot reducer astronbs
-```
+  * If you are interested in exoplanet light curve fitting, also  install `batman` via:
 
-## Running stellarphot
+    	pip install batman-package
```
Comment (mwcraig) [inline README.md:32]: they could also get this by doing `pip install stellarphot[exo_fitting]`

**Review by mwcraig — CHANGES_REQUESTED** [review]
Looks good, one change to make for sure, another you can take or leave.

File: README.md (hunk 2 above)
Comment (JuanCab) [reply README.md:32]: Fixed, since I think this is cleaner.

**Review by mwcraig — APPROVED** [review]

**Pull request merged**